src/main/java/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,11 @@
+package strategy;
+
+import core.basesyntax.FruitTransaction;
+import core.basesyntax.Storage;
+
+public class ReturnOperation implements OperationHandler {
```

Comment:
Following the SOLID principles, each class should have a single responsibility. The `ReturnOperation` class is responsible for handling the 'return' operation which is good. However, ensure that this class does not grow to handle multiple responsibilities in the future.